apps/meteor/app/discussion/client/discussionFromMessageBox.js

```diff
@@ -1,21 +1,26 @@
 import { Meteor } from 'meteor/meteor';
 import { Tracker } from 'meteor/tracker';
+import { Session } from 'meteor/session';
+import { isRoomFederated } from '@rocket.chat/core-typings';
 
 import { messageBox } from '../../ui-utils/client';
 import { settings } from '../../settings/client';
 import { hasPermission } from '../../authorization/client';
 import { imperativeModal } from '../../../client/lib/imperativeModal';
 import CreateDiscussion from '../../../client/components/CreateDiscussion/CreateDiscussion';
+import { Rooms } from '../../models/client';
 
 Meteor.startup(function () {
 	Tracker.autorun(() => {
 		if (!settings.get('Discussion_enabled')) {
 			return messageBox.actions.remove('Create_new', /start-discussion/);
 		}
+		const room = Rooms.findOne(Session.get('openedRoom'));
```

Comment:
This will load the room on startup, not when the action is clicked.

Reply:
my bad 😞